wiki/Gameplay/Judgement/osu!/en.md

```diff
@@ -1,57 +1,83 @@
 
 ### Sliders
 
 Sliders consist of multiple parts: the slider head, slider tail, slider ticks, and slider repeats. The slider as a whole is judged based on how many parts of the slider the player has hit, as outlined below:
 
-- 300: The player successfully hit all slider parts.
-- 100: The player hit at least half of the slider parts.
-- 50: The player managed to hit at least one of the parts.
-- Miss: The player did not successfully hit any of the slider parts.
-- If [ScoreV2](/wiki/Gameplay/Game_modifier/ScoreV2) is active, the accuracy of hitting the slider head is additionally accounted for when judging the entire slider:
-  - Getting a 300 for a slider requires getting a 300 judgement for the slider head.
-  - Getting a 100 for a slider requires getting a 100 judgement or better for the slider head.
+| Judgement | Slider completion |
+| :-: | :-- |
+| Great | 100% |
+| Ok | 50% |
+| Meh | At least one slider part |
+| Miss | 0% |
+
+The slider head only needs to be hit within the Meh hit window for a successful hit. However, if [ScoreV2](/wiki/Gameplay/Game_modifier/ScoreV2) is active, slider head accuracy is additionally accounted for, with the judgement received for the entire slider being limited to at most the one received for the slider head.
 
-The slider parts also have additional influence on the current [combo](/wiki/Gameplay/Combo_(score_multiplier)):
+There are some additional quirks with how sliders influence [combo](/wiki/Gameplay/Combo_(score_multiplier)):
 
-- Tapping the slider head too early does not incur a miss, but will reset combo to 0.
-- Missing a slider tick or repeat does not incur a miss, but will reset combo to 0.
-- Missing the slider end does not incur a miss, but will not increment combo.
+- Tapping the slider head too early (before the Meh hit window), missing a slider tick, or missing a repeat does not incur a Miss, but will cause a [combo break](/wiki/Gameplay/Judgement/Combobreak). The other slider parts can still be hit if a key is held down. This is colloquially referred to as a [slider break](/wiki/Gameplay/Judgement/Slider_break).
+- Missing the slider end does not incur a Miss, but will not increment combo.
 
 ### Spinners
 
-Each spinner has a set number of spins required to complete it. This number depends on the [overall difficulty](/wiki/Beatmap/Overall_difficulty#sliders-and-spinners) of the beatmap. The criteria for judging the spinner as a whole are as follows:
+Each spinner has a set number of spins required to complete it. This number depends on the [overall difficulty](/wiki/Beatmap/Overall_difficulty#sliders-and-spinners) of the beatmap.
+
+| Judgement | Spins required |
+| :-: | :-- |
+| Great | 100% |
+| Ok | One spin less than the required number |
+| Meh | 25% |
+| Miss | 0% |
+
+The required spins depends on the duration of the spinner and the overall difficulty. The spins per second required are given as follows:
+
+<!-- TODO: needs verification -->
```

Comment:
if you find out what "recent spinner changes" from https://github.com/ppy/osu-wiki/blob/master/wiki/Beatmap/Overall_difficulty/en.md?plain=1#L63 means, could you please add a reference to that information somewhere?

Reply:
if the values have changed, then there's no way to know except by verifying with a dev. do you mean mentioning the change in a history section or something?

Reply:
yeah, if you find a changelog entry, or anything that is public-facing

Reply:
turns out the release was already referred to in the history section, but i did modify it to add the stable changelog entry